google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Publisher.java

```diff
@@ -150,14 +151,24 @@
     static final long DEFAULT_MAX_BUNDLE_BYTES = 1000L; // 1 kB
     static final Duration DEFAULT_MAX_BUNDLE_DURATION = new Duration(1); // 1ms
```

Comment:
Done. MIN_{TOTAL,RPC}_TIMEOUT is kept unchanged, since they are actually the minimum we allow. The default just also happens to be the min.